60_network_learning.Rmd

```diff
@@ -0,0 +1,654 @@
+# Network learning and analysis {#network-learning}
+
```

Comment:
The intro chapter contains a lot of useful information. 

I am however thinking if we should get the user busy with recommended analysis faster, and have for instance in the end of this chapter a subsection that provides more details. Then the intro could be shortened, and it could cite that concluding section? 

I am mainly thinking about a user who just wants to get started with trying out some recommended approach, and then maybe check more methods and details after gaining some experience with one recommended method as a starting point?

Reply:
So, I've completely reorganized the chapter. The first sections now only show network learning with the SPRING method. And I added a section at the end, where the three types of association are compared.